spec/enumerable_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require File.expand_path('../enumerable_methods', __dir__)
+
```

Comment:
Kindly change this to `require_relative`. It is Ruby convention to use require_relative within a directory.

Reply:
`require_relative '../enumerable_methods'`